tests/indexes/models.py

```diff
@@ -26,7 +26,8 @@ class ArticleTranslation(models.Model):
 
 class Article(models.Model):
     headline = models.CharField(max_length=100)
-    pub_date = models.DateTimeField()
+    pub_date = models.DateTimeField(null=True)
```

Comment:
I'm not sure why I added `null=True`, so it should probably be removed.

Reply:
what about auto_now=True instead of null=True?

Reply:
The tests pass without any change to that line, so I'll just revert my addition, and don't plan on changing it again.